src/modules/media/types.ts

```diff
@@ -120,3 +99,25 @@ export interface Media {
 	ebucoreObjectType: string;
 	representations: Representation[];
 }
+
+export enum SearchFilterField {
+	QUERY = 'query',
+	ADVANCED_QUERY = 'advancedQuery',
+	FORMAT = 'format',
+	DURATION = 'duration',
+	CREATED = 'created',
+	PUBLISHED = 'published',
+	CREATOR = 'creator',
+	GENRE = 'genre',
+	KEYWORD = 'keyword',
+	NAME = 'name',
+}
+
+export enum Operator {
+	CONTAINS = 'contains',
+	CONTAINS_NOT = 'containsNot',
+	IS = 'is',
+	IS_NOT = 'isNot',
+	GTE = 'gte',
+	LTE = 'lte',
```

Comment:
Question: So the `between 2 dates` filter will be 2 entries in the filter array, one with operator GTE and one with LTE? ok, that is pretty clean, then we don't need to handle the `if one is set and the other isn't logic`

Reply:
Correct!